docs/ios-sdk.md

```diff
@@ -1,6 +1,6 @@
 # Livery iOS SDK
 
-Ex Machina Group Livery iOS SDK, published as CocoaPods spec on Github: [livery-sdk-ios-podspec](https://github.com/exmg/livery-sdk-ios-podspec).
+Livery iOS SDK, published as CocoaPods spec on Github: [livery-sdk-ios-podspec](https://github.com/exmg/livery-sdk-ios-podspec).
 
 Documentation can be found at: [docs.liveryvideo.com/ios-sdk](https://docs.liveryvideo.com/ios-sdk).
 
```

Comment:
The Livery iOS SDK is compatible with iOS 12 **of** higher.

I think that should be iOS 12 **or** higher right?